octoprint_mrbeam/static/js/app/view-models/design-store.js

```diff
@@ -162,6 +170,13 @@ $(function () {
             };
 
             self.sendMessageToDesignStoreIframe("userData", userData);
+
+            if (self.laserheadChangedVM.isLaserheadChanged()) {
```

Comment:
I think we only want to send the event if the laserhead x is the one selected so far right?

Reply:
Yea the idea was to keep the handling of the detected laser head to the design store if we decided to also tag S laser head designs or for any future laser heads; However we can limit it to the X laser head for now.

Reply:
done